tests/foreman/cli/test_contenthost.py

```diff
@@ -567,115 +559,38 @@ def setUp(self):
 
```

Comment:
Yes, Sorry I didn't mention setUpClass explicity!  But I thought of just using setUpClass and move vm creation part to the tests or perhaps as an other small function which the tests can call.  But your approach of using setUpClass and setUp looks nice as well.

@elyezer What do you think about this?